fix/basics-cho-api (#2318)
mercadopago/devsite-docs: guides/additional-content/sales-processing/go-live-requirements.pt.md

# Requisitos para entrar em produção

Ao finalizar o processo de integração, o ambiente estará pronto para ser colocado em produção. Nesta documentação, detalharemos os requisitos necessário para realizar essa mudança de maneira eficaz e segura, garantindo que a integração esteja preparada para o recebimento de pagamentos reais.

## Ativar credenciais de produção

Para começar a receber pagamentos, você deve **ativar as credenciais de produção** e substituir as credenciais de teste. 

> Para isso, acesse o [Painel do Desenvolvedor](https://www.mercadopago[FAKER][URL][DOMAIN]/settings/account/credentials) e, no menu lateral, acesse **Produção > Credenciais de Produção**. Ali você encontrará seu _Public Key_ y _Access Token_ produtivos, que devem substituir os de teste utilizados nas etapas anteriores.

Author: ```suggestion
Para isso, acesse o [Painel do Desenvolvedor](https://www.mercadopago[FAKER][URL][DOMAIN]/settings/account/credentials) e, no menu lateral, acesse **Produção > Credenciais de Produção**. Ali você encontrará seu _Public Key_ e _Access Token_ produtivos, que devem substituir os de teste utilizados nas etapas anteriores.
```

Quando sua integração estiver pronta e você quiser começar a receber pagamentos, você deve [ativar as credenciais](/developers/pt/guides/additional-content/your-integrations/credentials) de produção e substituir as credenciais de teste, se necessário. 

Além disso,  recomendamos que você cumpra as seguintes considerações adicionais.

## Homologação

> Para oferecer a melhor experiência tanto para o vendedor quanto para o comprador, é importante avaliar a [qualidade de sua integração](/developers/pt/guides/additional-content/homologator/homologator) com nossa ferramenta de homologação. Este processo possibilita certificar-se de que a integração cumpre com os requisitos de qualidade e segurança necessários.  

Reviewer: @pcanosa - Pili, conforme falamos hoje, aqui vale colocar um pouco mais de info sobre a ferramenta e sua importância.

## Certificado SSL

> Para que sua integração seja segura e proteja os dados das transações, **é necessário ter um certificado SSL e que a forma de pagamento seja disponibilizada em uma página HTTPS**. 

Reviewer: ```suggestion
Para garantir uma integração segura e que proteja os dados de cada transação, é necessário implementar um certificado SSL. Além disso, é importante que a forma de pagamento seja disponibilizada em uma página web que utilize o protocolo HTTPS. Este protocolo assegura a criptografia dos dados transmitidos entre o cliente e o servidor.
```

> Com isto, é possível garantir a segurança dos dados de seus clientes, cumprir os regulamentos ou disposições legais de cada país e conseguir a melhor experiência de compra para suas vendas.

Reviewer: ```suggestion
Adotar estas medidas não apenas reforça a segurança dos dados dos usuários, mas também assegura a conformidade com as normativas e leis específicas de cada país relativas à proteção de dados e à segurança da informação. Além disso, contribui significativamente para proporcionar uma experiência de compra mais segura e confiável.
```

> Apesar de não ser obrigatório durante a fase de testes, o certificado é obrigatório para iniciar em produção. Para mais informações, [confira os termos e condições do Mercado Pago](/developers/pt/guides/resources/legal/terms-and-conditions).

Reviewer: ```suggestion
Embora a exigência do certificado SSL não se aplique durante o período de testes, sua implementação é obrigatória para entrar em produção.. Para mais informações, [confira os termos e condições do Mercado Pago](/developers/pt/guides/resources/legal/terms-and-conditions).
```

Reviewer: @pcanosa - acho que isso pode ser um warning laranja. Além disso, fui ler os termos e condições e não vi nada que fala sobre esse certificado. Você achou?

----[mlb]----
## Chave Pix

A opção de pagamento com Pix só será exibida se houver uma chave Pix cadastrada no Mercado Pago. Caso ainda não tenha criado, [clique aqui](https://www.youtube.com/watch?v=60tApKYVnkA) e veja o passo a passo.

------------

## Aprovação de pagamentos

> Saiba quais medidas você pode tomar para melhorar a [aprovação de pagamentos](/developers/pt/guides/additional-content/how-tos/payment-rejections), como o envio de informações do item e do pagador, dados de envio e informações da indústria, entre outros.

Reviewer: ```suggestion
Para otimizar a taxa de aprovação de pagamentos, diversas estratégias podem ser adotadas, abrangendo desde a inclusão precisa de informações relacionadas ao item vendido e aos dados do comprador, até a entrega de detalhes sobre o envio e especificidades do setor de atuação. 

Para mais informações sobre os principais motivos pelos quais um pagamento pode ser recusado e como fazer para evitar essas recusas, acesse a documentação [Como melhorar a aprovação de pagamentos](/developers/pt/guides/additional-content/how-tos/payment-rejections)
```

> ## Notificações

Reviewer: @pcanosa - Pili, faz sentido falarmos de notificação como um requisito para entrar em produção? O que acha?
